src/events/aws_sdk_v3.js

```diff
@@ -159,12 +165,10 @@ module.exports = {
      */
     init() {
         moduleUtils.patchModule(
-            '@aws-sdk/client-sns',
+            '@aws-sdk/smithy-client',
```

Comment:
pls add a comment for those stupid people like me who don't know what smithy-client is :)